src/mobile/screens/ProfileAuth/index.tsx

```diff
@@ -0,0 +1,110 @@
+import cr from 'classnames';
+import * as React from 'react';
+import { Button } from 'react-bootstrap';
+import { useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router';
+import { CustomInput } from '../../../components/CustomInput';
+import { ProfileTwoFactorAuth } from '../../../containers/ProfileTwoFactorAuth';
+import { selectUserInfo, toggle2faFetch } from '../../../modules/user/profile';
+import { ProfileTwoFactorAuthScreen } from '../../../screens/ProfileTwoFactorAuthScreen';
+import { Modal } from '../../components/Modal';
+import { Subheader } from '../../components/Subheader';
+
+export const ProfileAuthMobileScreen = () => {
```

Comment:
Add type to componets. In this case:

    export const ProfileAuthMobileScreen: React.FC = () => {